roundedimageview/src/main/java/com/makeramen/roundedimageview/Corners.java

```diff
@@ -0,0 +1,78 @@
+package com.makeramen.roundedimageview;
+
+public enum Corners {
+
+  ALL(1, 1, 1, 1),
+  TOP_LEFT(1, 0, 0, 0),
+  TOP_RIGHT(0, 1, 0, 0),
+  BOTTOM_LEFT(0, 0, 1, 0),
+  BOTTOM_RIGHT(0, 0, 0, 1),
+  TOP(1, 1, 0, 0),
+  BOTTOM(0, 0, 1, 1),
+  LEFT(1, 0, 1, 0),
+  RIGHT(0, 1, 0, 1),
+  OTHER_TOP_LEFT(0, 1, 1, 1),
```

Comment:
I'm not sure this is a better API, and would be a breaking change and require a major version bump.

EnumSets or bitmasks are probably better solutions here.